generators/app/templates/7/_wercker.yml

```diff
@@ -88,10 +78,6 @@ build:
             drush @<%= appName %>.<%= appName %>.dev ups | grep -qi "update available" && (echo 'Updates available FAIL' && exit 1) || (echo 'No updates available PASS' && exit 0)
             # Behat tests.
             ./.vendor/bin/behat -p wercker
-            # Visual regression tests.
-            # cd wraith && wraith capture capture
-            # Store wraith shots in wercker artifacts folder.
-            # cp -r shots $WERCKER_REPORT_ARTIFACTS_DIR/
```

Comment:
We're not testing backstop in wercker. Is that intended?
I'd create a new step for the backstop tests